dbt/adapters/athena/impl.py

```diff
@@ -57,6 +57,73 @@ def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
     def convert_datetime_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "timestamp"
 
+    def _parse_lf_tags(self, lf_tags_expr: Optional[str]) -> Dict[str, str]:
+        lf_tags = {}
+        if lf_tags_expr:
+            lf_tags = dict([tag.split("=") for tag in lf_tags_expr.split(",")])
```

Comment:
why not passing directly lf_tags as a dictionary `{'origin': 'dbt', 'team': 'analytics'} ` I think that is a bit more error prone for the user. We use a similar setup for the extra table properties of iceberg.

Reply:
@nicor88 that makes a ton of sense, updated 👍